src/models/supplemental_accessors.jl

```diff
@@ -27,13 +41,20 @@ function get_series_susceptance(b::Union{TapTransformer, PhaseShiftingTransforme
     return y_a
 end
 
+function get_series_susceptance(::Union{PhaseShiftingTransformer3W, Transformer3W})
```

Comment:
@m-bossart, @mcllerena  and @luke-kiernan take a look that this won't break stuff in PNM and PFS. Having two methods return different types was a bad idea.